compiler/runtime/OMRCodeCache.cpp

```diff
@@ -587,6 +587,11 @@ OMR::CodeCache::findTrampoline(TR_OpaqueMethodBlock * method)
          {
          void *newPC = (void *) TR::Compiler->mtd.startPC(method);
 
+         if (reinterpret_cast<intptr_t>(newPC) <= 0)
```

Comment:
this doesn't look right.
if there is no existing trampoline, the _currentStartPC (associated with the trampoline) must be NULL. it didn't mean the method hasn't been compiled though.

Reply:
I think you are right. I'll need to take a look at this again.